extensions/amp-jwplayer/amp-jwplayer.md

```diff
@@ -91,19 +91,11 @@ Example:
   </tr>
   <tr>
     <td width="40%"><strong>data-content-search</strong></td>
-    <td>Denotes the type of the playlist. This is a search playlist that takes in a keyword or phrase as the search query and generates a playlist based on that search query. If contextual article matching is desired, use the value <code>__CONTEXTUAL__</code> (data-content-contextual must also be <code>true</code>).)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-content-contextual</strong></td>
-    <td>Enables the Player to grab the OG title (or HTML title if there is no OG title) of a given webpage and use that as the search query (required to do Contextual Article Matching). (<strong>'true' or 'false'</strong>)</td>
-  </tr>
-  <tr>
-    <td width="40%"><strong>data-content-recency</strong></td>
-    <td>Limits the videos added into the playlist based on their age (i.e. the playlist includes videos that are max [xx] days old). Specified in the format <code>xD</code>, where x is a numerical value (i.e. <code>'9D'</code>).</td>
+    <td>Denotes the type of the playlist. If contextual article matching is desired, use the value <code>`__CONTEXTUAL__`</code>. If a search playlist is desired, input a keyword or phrase used to generate the search playlist.</td>
```

Comment:
How does `__CONTEXTUAL__` still apply?

Reply:
The use of `__CONTEXTUAL__` is unchanged and should be passed back through `data-content-search`.  Previously, we required an additional field `data-content-contextual=TRUE` concurrently to enable the feature.  This additional requirement has now been removed for simplicity.